ted_sws/data_sampler/entrypoints/notebooks/notice_sampler_runner.ipynb

```diff
@@ -1,1663 +0,0 @@
-{
- "cells": [
-  {
-   "cell_type": "code",
```

Comment:
don't think is a good idea to delete this notebook

Reply:
the notebooks were moved to mapping-toolchain project (in case of critical need, can be moved back)